app/components/LoadMore.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { useInView } from "react-intersection-observer";
+import Spinner from './Spinner';
+import Post from "./Post";
+
+/*TODO: Instead of passing in postData as a Prop to this component, this comp.
+ * should call an API route that returns the posts.
+ */
+export default function LoadMore({postsData}: {postsData: Object[]}) {
+  // This state keeps track of post data
```

Comment:
```suggestion
```
Unnecessary comment, it's inferred this use of `useState` deals with posts because of the name of the state